tests/scripts/github-action-helper.sh

```diff
@@ -133,7 +133,10 @@ function validate_yaml() {
   cd cluster/examples/kubernetes/ceph
   kubectl create -f crds.yaml -f common.yaml
   # skipping folders and some yamls that are only for openshift.
-  kubectl create $(ls -I scc.yaml -I "*-openshift.yaml" -I "*.sh" -I "*.py" -p | grep -v / | awk ' { print " -f " $1 } ') --dry-run
+  manifests="$(find . -maxdepth 1 -type f -name '*.yaml' -and -not -name '*openshift*' -and -not -name 'scc*')"
```

Comment:
Is `maxdepth 1` necessary since we are in `cluster/examples/kubernetes/ceph` already?

Reply:
```suggestion
  manifests="$(find . -maxdepth 1 -type f -name '*.yaml' -and -not -name '*openshift.yaml' -and -not -name scc.yaml)"
```

Reply:
without `maxdepth 1` it was fetching `monitoring/*` files also